buildprocess/generateCatalogIndex.ts

```diff
@@ -136,13 +136,15 @@ export default async function generateCatalogIndex(
           { expiration: 30000, priority },
           async () => {
             console.log(`Loading Reference ${name} (${path}) = ${priority}`);
-            const result = await (member as ReferenceMixin.Instance).loadReference();
+            const result = await (
+              member as ReferenceMixin.Instance
+            ).loadReference();
             result.logError(`FAILED to load Reference ${name} (${path})`);
             result.pushErrorTo(
               errors,
               `FAILED to load Reference ${name} (${path})`
             );
-            result.catchError(e => console.error(e.toError().message));
+            result.catchError((e) => console.error(e.toError().message));
```

Comment:
Is it possible to configure prettier to keep single argument anon functions without brackets?

Reply:
Yes setting `"arrowParens": "avoid"` in `.prettierrc` will do that, I personally prefer to always add parens as it is easier to add type annotation, another param, consistency and bunch of other options, I also believe most of the developers are used to this new syntax. but in the end prettier handles them so perfectly fine with both

https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev

Reply:
Yep fair enough, I think you're right.